### PR TITLE
Updated GEMPAK version and APRUN launcher.

### LINKED ISSUE #2248: Wrong launcher command for Gempak jobs
### What is wrong?

The wrong launcher is being specified by `JGDAS_ATMOS_GEMPAK`:

https://github.com/NOAA-EMC/global-workflow/blob/f4d187f4e45fe89583d18987d68a883490827104/jobs/JGDAS_ATMOS_GEMPAK#L80-L84

Since `APRUN_GEMPAKCFP` is not declared in the `env` file (`APRUN` is), `mpiexec` is called, which fails on Hera as `--cpu-bind` is not an available option for the version on that machine.

### What should have happened?

The `HERA.env` file should have specified `APRUN_GEMPAKCFP`.  This is true for the other systems as well.

### What machines are impacted?

All or N/A

### Steps to reproduce

Run a cycled forecast with `DO_GEMPAK=YES` and allow the `gempak` jobs to execute, then check the logs.

### Additional information

N/A

### Do you have a proposed solution?

Add `APRUN_GEMPAKCFP` to each `env` file to specify the correct launcher for `CFP` programs.

### LINKED ISSUE #2513: Update gempak version
### What new functionality do you need?

Need to update gempak version on Hera to `7.17.0` so `7.4.2` can be removed.

### What are the requirements for the new functionality?

Gempak on Hera runs when loading 7.17.0 version

### Acceptance Criteria

- [ ] Gempak runs loading 7.17.0 at runtime
- [ ] All products are produced as current (may not be bitwise compatible?)

### Suggest a solution (optional)

_No response_

### Dependencies
- [ ] https://github.com/NOAA-EMC/gfs-utils/issues/68

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
# Description

This PR addresses issue #2248 and #2513. The following is accomplished:

- Updates the parallel executable for GEMPAK applications;
- Updates the GEMPAK version for RDHPCS Hera.

  Describe your changes. Focus on the *what* and *why*. The *how* will be evident from the changes. In particular, be sure to note any interface changes, such as command line syntax, that will need to be communicated to users.

  At the end of your description, please be sure to add the issue this PR solves using the word "Resolves". If there are any issues that are related but not yet resolved (including in other repos), you may use "Refs".

  Resolves #2248 
  Resolves #2513 

# Type of change

- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

Tested on RDHPCS Hera. The following is the Rocoto status:

```
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202103231200               gfsgempak                    58973651           SUCCEEDED                   0         1         526.0
```

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by HenryRWinterbottom** [comment]
@DavidHuber-NOAA This addresses only RDHPCS Hera. I want to make sure this addresses all points in #2248 before I update the remaining platforms.

**Comment by WalterKolczynski-NOAA** [comment]
There should be more gempak jobs than just one.

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]
@HenryWinterbottom-NOAA It looks like this was indirectly fixed by @WalterKolczynski-NOAA in #2374.  The GEMPAK jobs now call `ush/run_mpmd.sh` to run their MPMD jobs.  Within run_mpmd.sh, it keys off of `launcher` instead of `APRUN` (or any variety of `APRUN*`).

So instead, I think the correct action is to remove `APRUN` definitions from all of the `env` files.

File: env/HERA.env [hunk 1]
```diff
@@ -296,7 +296,7 @@ elif [[ "${step}" = "gempak" ]]; then
 
     export NTHREADS_GEMPAK=${nth_gempak:-1}
     [[ ${NTHREADS_GEMPAK} -gt ${nth_max} ]] && export NTHREADS_GEMPAK=${nth_max}
-    export APRUN="${launcher} -n ${npe_gempak} ${mpmd_opt}"
+    export APRUN_GEMPAKCFP="${launcher} -n ${npe_gempak} ${mpmd_opt}"
```
Comment (DavidHuber-NOAA) [inline env/HERA.env]:
```suggestion
```

**Comment by DavidHuber-NOAA** [comment]
To clarify, by "remove all of the `APRUN` definitions`, I'm referring to just the GEMPAK job.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good, thanks!

**Pull request merged**